### PR TITLE
[master] Avoid dots, full path to a file.

### PR DESCRIPTION
**Pull request by mozga-intel**
## Description ##
This pull request contains changes that included the full path to a file. This pull-request contains only oneDNN dirs.

### Essentials ###
- [ ] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by mozga-intel** [comment]
@mxnet-bot run ci [centos-gpu, unix-cpu, website, windows-gpu]

**Comment by mozga-intel** [comment]
@mxnet-bot run ci [unix-gpu, windows-cpu]

**Comment by mozga-intel** [comment]
@mxnet-bot run ci [unix-cpu]

**Comment by mozga-intel** [comment]
@mxnet-bot run ci [unix-cpu, windows-gpu]

**Comment by mozga-intel** [comment]
@mxnet-bot run ci [unix-cpu]

File: src/operator/nn/dnnl/dnnl_base.cc [hunk 1]
```diff
@@ -22,7 +22,7 @@
 #include <atomic>
 
 #include "../../../common/exec_utils.h"
```
Comment (bartekkuncer) [inline src/operator/nn/dnnl/dnnl_base.cc:24]: Why this one has been left unchanged?

Comment (mozga-intel) [reply src/operator/nn/dnnl/dnnl_base.cc:24]: This file comes from another scope ~ there I have another idea.

**Review by DominikaJedynak — APPROVED** [review]
LGTM, please delete ./ to make it more readable :)

**Comment by mozga-intel** [comment]
@mxnet-bot run ci [unix-cpu, windows-gpu]

**Comment by mozga-intel** [comment]
@mxnet-bot run ci [unix-cpu, unix-gpu, windows-gpu, centos-cpu]

**Comment by mozga-intel** [comment]
@mxnet-bot run ci [windows-gpu]

**Review by bartekkuncer — APPROVED** [review]
LGTM

**Review by bgawrych — APPROVED** [review]

**Comment by mozga-intel** [comment]
@mxnet-bot run ci [unix-cpuunix-cpuunix-cpuunix-cpuunix-cpuunix-cpu]

**Comment by mozga-intel** [comment]
@mxnet-bot run ci [unix-cpu]

**Pull request merged**